### PR TITLE
CCloudify offset partition

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

* Adds Confluent button for this KT
* Adds short answer

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/convert_offset_partition/kafka-console-consumer-read-specific-offsets-partitions/confluent.html

### CODE REVIEW

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/produce-topic-keys-values.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
+To get started, lets produce some records to your new topic.
+
+Since you've created a topic with more than one partition, you'll send full key-value pairs so you'll be able to see how different keys end up on different partitions.
+To send full key-value pairs you'll specify the `parse.key` and `key.separator` options to the console producer command.
+
+Let's run the following command in the broker container shell from the previous step to start a new console producer:
```
Comment (awalther28) [inline _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/produce-topic-keys-values.adoc:6]: At this point we don't have any containers running so "broker container shell" should probably be changed to just CCloud CLI producer. Same with on line 19, talking about the broker container.

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-offsets.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/confluent/code/tutorial-steps/dev/console-consumer-keys-partition-offset.sh %}</code></pre>
++++++
+
+After a few seconds you should see something like this
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-read-specific-offsets-partition/confluent/code/tutorial-steps/dev/expected-output-step-three.txt %}</code></pre>
++++++
+
+So you can see here, you've consumed records starting from offset 3 to the end, which includes record with offsets of `3, 4, and 5` the last three records.
```
Comment (awalther28) [inline _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-offsets.adoc:20]:
```suggestion
So you can see here, you've consumed records starting from offset `3` to the end, which includes records with offsets of `3`, `4`, and `5` (the last three records).
```
I can see this is language carried over from the original. Maybe it's a me problem but I'm having a hard time understanding what this is conveying. I'm not sure if `and` needs to be in code. Feel free to disregard if you think the original is fine as is!

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Next let's open up a console consumer to read records sent to the topic in the previous step, but you'll only read from the first partition. Kafka partitions
+are zero based so your two partitions are numbered `0`, and `1` respectively.
+
+Confluent Cloud CLI currently does not have the ability to specify reading from a particular partition, so for the next few steps you'll be using the console consumer built into a Docker image.
+Notice that you are passing in the path to the `configuration/ccloud.properties` file that you created earlier.
```
Comment (awalther28) [inline _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc:5]: Maybe not a big deal but at this point, the user hasn't created a `configuration` directory.

**Review by awalther28 — COMMENTED** [review]
Awesome, so excited to see another conversion! Thanks @ybyzek!

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-offsets.adoc (hunk 2 above)
Comment (ybyzek) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-offsets.adoc:20]: @awalther28 agree, it's confusing.  I've wordsmith'd both versions in latest commit.

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc (hunk 3 above)
Comment (ybyzek) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc:5]: This is actually true for all CCloud KTs.

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/produce-topic-keys-values.adoc (hunk 1 above)
Comment (ybyzek) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/produce-topic-keys-values.adoc]: Good catch, fixed in latest commit

File: _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc (hunk 3 above)
Comment (awalther28) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc:5]: Not the ones I've done :) checkout out [cogrouping](https://kafka-tutorials.confluent.io/cogrouping-streams/confluent.html#initialize-the-project ) or [filtering](https://kafka-tutorials.confluent.io/filter-a-stream-of-events/confluent.html#initialize-the-project) as examples. You don't necessarily have to do add that step.

Comment (ybyzek) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc:5]: 🙇 I will fix ;)

Comment (ybyzek) [reply _includes/tutorials/console-consumer-read-specific-offsets-partition/confluent/markup/dev/consume-topic-partition-zero.adoc:5]: Done in latest commit

**Review by awalther28 — APPROVED** [review]
One more close to our goal, thanks @ybyzek ! Feel free to merge this!

**Comment by ybyzek** [comment]
Unrelated test failure

**Pull request merged**